tests/e2e/notifications/messages/conftest.py

```
@@ -0,0 +1,25 @@
+        },
+        "externalIds": {"test": f"e2e-delete-{short_uuid}"},
+    }
+
+
+@pytest.fixture
+def message_id(e2e_config):
+    return e2e_config.get("notifications.message.id")
+
+
+@pytest.fixture
+def invalid_message_id(e2e_config):
```

Ruff (0.14.7)

Unused function argument: `e2e_config`

(ARG001)